akka-actor/src/main/resources/reference.conf

```diff
@@ -652,7 +652,7 @@ akka {
     # `akka.actor.serialization-identifiers."FQCN" = ID`
     # where `FQCN` is fully qualified class name of the serializer implementation
     # and `ID` is globally unique serializer identifier number.
-    # Identifier values from 0 to 16 are reserved for Akka internal usage.
+    # Identifier values from 0 to 22 are reserved for Akka internal usage.
```

Comment:
We should probably set a higher number to make space for possible future ones also. 40 sounds like a good arbitrary number to me.